### ISSUE TITLE
Bibliography needs to be unified and checked again

### ISSUE THREAD

**Issue by LinguList** [lexibank/chacolanguages#13]
@Bridnicolas, the bibliography needs to be unified with respect to the style (hyperlinks in the same color, etc.) but also with respect to the format, like Author names, first lastname then first name, etc., all authors to be spelled, DOI, where available. And there are three items missing. Can I ask you to first check this, again, that we do not miss references, and then inform me, so I have a look?

**Comment by Bridnicolas** [comment]
No need to answer. I saw the guidelines already. Unnecessary question.

**Comment by LinguList** [comment]
For the journal, according to my knowledge, any format that is 
consistent is okay. So there should not be different variants, that 
would be all ;)

**Comment by Bridnicolas** [comment]
It's mostly done. Only Vidal's dictionary is lacking, because I really don't know how to cite it (no one else did and it's a we-transfer). I'll ask Cristina about that. You may see some comments with tiny questions in the bibliography.

**Comment by LinguList** [comment]
But the dictionary has a title sheet?

**Comment by LinguList** [comment]
@Bridnicolas, please make sure to fix these when you are back.

**Comment by Bridnicolas** [comment]
Mattis,

Be sure I'll see to it when I'm back. I'll hopefully be in Asunción next
Sunday. I'll be informing you then. Sorry I didn't answer before, I had no
internet.

Best,

Nicolás

**Comment by FredericBlum** [comment]
@Bridnicolas Could you please check this issue and correct the sources, maybe at some point during this or next week? We are currently preparing for a new release of all Lexibank datasets and it would be good to have this resolved.

**Comment by Bridnicolas** [comment]
@FredericBlum Yes, sure. I'll probably have to time look into it deeply during the weekend.

**Comment by FredericBlum** [comment]
@Bridnicolas Any update on this?

**Comment by Bridnicolas** [comment]
@FredericBlum Sorry, not yet. I'm in the crazy middle of writing a review, an article, a scholarship application, and moving in. I'm committed to this, too, but I'd like to know what the deadline is, so I can arrange my time.

**Comment by FredericBlum** [comment]
Would be neat to have this by the end of next week. Shouldn't be too much work, considering it is only the four problems mentioned above.

**Comment by Bridnicolas** [comment]
Great. I did consider it big work, tbh, but just out of sheer ignorance. I'll text you back one of these days (this week), when I get down to it.

**Comment by FredericBlum** [comment]
@Bridnicolas Any updates?

**Comment by Bridnicolas** [comment]
@FredericBlum I'm moving things to my new house right now, if it's not much of a problem I'll send you updates in the afternoon.

**Comment by Bridnicolas** [comment]
@FredericBlum Sorry for the delay. Machoni (1877) is correct now, according to the link @LinguList provided above. It was first printed in Madrid in 1732, and then in BsAs in 1877.
I've added the address on Kalisch & Unruh 1997.
As for Kalisch 2021, I must have been confused with the IDS online version. Hannes didn't write any dictionary in 2021. We can do away with that reference.
All the IDS online versions are from 2015, because that's how it appeared on IDS. I can correct that too if it's wrong.
As for Buckwalter Mocoví's dictionary, the 2004 version I used was of internal circulation, not found on the web, and digitalized by a local Secretary of Environment (?). I'll ask Cristian if he knows how to cite it properly. 
I'm not sure if I understood the task properly. If not, please let me know and I'll revise the bibliography again.

**Comment by FredericBlum** [comment]
Great, thanks, then all we need is the reference for Buckwalter and then we can go ahead and update :)

**Comment by FredericBlum** [comment]
@Bridnicolas Could you solve the problem about the Mocovi dictionary?

**Comment by Bridnicolas** [comment]
I think so. Cristian guided me into it. For the time being I'd rather eliminate that rare 2004 source and stick to the IDS Buckwalter Mocoví wordlist, which is both more reliable (Cristian told me) and citable. I used this rare source in the beginning, but I doubt that the data will be significantly different from the IDS wordlist, because it's basic vocab. Unless you want otherwise, 
I already committed the changes, having also changed some more details I found there. Btw, where I should be making changes is on sources.bib, right?
